shell/platform/android/test/io/flutter/embedding/android/FlutterAndroidComponentTest.java

```diff
@@ -272,6 +272,15 @@ public boolean shouldAttachEngineToActivity() {
       return true;
     }
 
+    @Override
+    public void onFlutterSurfaceViewCreated(@NonNull FlutterSurfaceView flutterSurfaceView) {}
```

Comment:
Can we add something here that trakcs that this was called and assert it got called with a valid object?

Reply:
Tests added to ensure callbacks are invoked.